### DISCUSSION TITLE
Simulation Runs on Rstudio cloud but fails when run locally

### DISCUSSION THREAD

**Discussion by Jeankwizera** [RangeShifter/RangeshiftR-tutorials#26] [General] [answered]
I am experiencing a challenge with simulating dispersal using rangeshiftR I would like some assistance to understand this. 

I set up a simulation in "Rstudio Cloud" and the experiment runs successfully and returns the "results" I need. 

However, the exact experiment fails with the following error when I run it in "Rstudio locally":

___________________________________
Run Simulation(s) with seed 13976 ...
LandscapeFile headers OK: /Users/ ... /Inputs/climate_suitability.asc
Species Distribution map headers OK: /Users/ ... /Inputs/news_location.asc
At (x,y) = 14,23 :
*** Corrupted file stream in /Users/kwizera.jvk/Desktop/KWIZCONSULT/Range dispersal simulation/Inputs/climate_suitability.asc
Too few entries? Unsuppoerted file encoding? (You might try to use a different one, like UTF-8.)

Error reading landscape 0 - aborting
_______________________________

This is the link to the "Rstudio cloud project" for assistance.
https://rstudio.cloud/project/4019030

I am working with : R version 4.2.0 (2022-04-22) -- "Vigorous Calisthenics"

and:
RStudio 2022.02.2+485 "Prairie Trillium" Release (8acbd38b0d4ca3c86c570cf4112a8180c48cc6fb, 2022-04-19) for macOS
Mozilla/5.0 (Macintosh; Intel Mac OS X 11_3_1) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.10 Chrome/69.0.3497.128 Safari/537.36

I would like to be able to run the experiment on my local machine as well without the error. Thankyou

**Comment by damariszurell** [comment, marked as answer]
Hi there,
When loading and writing the climate suitability to file you use probabilities ranging 0-1. However, RangeShiftR expects percentage value 0-100%. Thus, you code in line 44 should look like this:

raster::writeRaster(round(AllData2a*100), format="ascii", filename = "Inputs/climate_suitabilitya", NAflag = -9, overwrite = T, bylayer = T, datatype = "INT2U")

Best,
Damaris

**Reply by Jeankwizera** [reply]
Hi, this is really helpful. Thankyou @damariszurell
